bg_simulator.py

```diff
@@ -58,7 +55,7 @@ def assign_carbs(sugar, last_carbs, sugar_in_range):
         carbs = random.triangular(-50, 100, 60)
     return carbs
 
-def stitch_func(num_days=180):
+def simulate(num_days=10):
```

Comment:
The other place is the 'num-days' argparse argument. I think I have to keep it, because there needs to be a default which is being passed in unless the user changes it and for now the default is 10, but once the project is done ill switch it back to 180. I could also just remove the default from the simulate function and just have it as num_days and keep the num-days argparse argument in place?